go-msgpack/codec/ext_dep_test.go

```diff
@@ -1,4 +1,5 @@
-//+build ignore
+//go:build ignore
+// +build ignore
```

Comment:
can't we just get rid of this line (line 2) here?

Reply:
We might want to keep it around for just a bit to support Go versions before go1.17. But I think you're right that for Consul we won't need it since we are already using go1.17.  I didn't add this myself, `goimports` added it when I ran it.